src/widget/wtracktableview.cpp

```diff
@@ -1775,6 +1780,17 @@ void WTrackTableView::doSortByColumn(int headerSection) {
     horizontalScrollBar()->setValue(savedHScrollBarPos);
 }
 
+void WTrackTableView::applySorting() {
+    int sortColumn = (int)m_pSortColumn->get();
```

Comment:
we prefer to use static_cast<int>() or just remove the cast.

Reply:
There are some other places in this PR.

Reply:
Fixed via 939ca38.